helixbass/tree-sitter-grep
# More languages

In this PR:
- add a bunch of other languages

Closes #14 

It looks like the release binary size went from ~6M to ~24M with these additional language crates added, I guess that was more or less to be expected and doesn't seem that large for a binary executable?

To test:
If you want to try running queries against any of the newly added languages they should work as expected/symmetrically to existing behavior
You should be able to get a "`*.h` is ambiguous" error if you pass a query that could be parsed as more than one of a C/C++/Objective-C query in auto-language mode in a directory containing a `*.h` file
Conversely, if you pass a query that eg would only parse for Objective-C in the above scenario, it should run that query as an Objective-C query against the `*.h` files

Based on `ignore-match-metadata`
